api/agent/agent.go

```diff
@@ -602,6 +618,14 @@ func (s *hotSlot) exec(ctx context.Context, call *call) error {
 	}
 }
 
+// Golang doesn't have an integer min...
+func min(a uint64, b uint64) uint64 {
+	if a < b {
+		return a
+	}
+	return b
+}
+
```

Comment:
some of these functions are in resource.go (we might want to move them to a common.go or utils.go or something like that.) Not important. :-)

Reply:
`int(math.Min(float64(a), float64(b)))` _usually_ works, float precision can get dicey but I doubt it for the numbers we're inputting. bit gross